CommonLibrary.cmake

```diff
@@ -54,12 +55,12 @@ macro(GENERATE_PROJECT_HEADER NAME)
         PUBLIC_HEADER MATCHES ".*\\.hpp$"))
 
       file(APPEND ${PROJECT_GENERATED_HEADER}
-        "#include <${PROJECT_INCLUDE_NAME}/${PUBLIC_HEADER}>\n")
+        "#include <${INCLUDE_NAME}/${PUBLIC_HEADER}>\n")
     endif()
   endforeach()
   file(APPEND ${PROJECT_GENERATED_HEADER} "#endif\n")
   set(PROJECT_INCLUDE_HEADER
-    ${PROJECT_BINARY_DIR}/include/${PROJECT_INCLUDE_NAME}/${BASE_NAME}.h)
+    ${OUTPUT_INCLUDE_DIR}/${INCLUDE_NAME}/${BASE_NAME}.h)
 
   configure_file(
```

Comment:
Good point. I don't know why. I accidentally inherited this bad code :) I'll fix it and test.